cumulusci/cli/cci.py

```diff
@@ -958,10 +967,11 @@ def org_list(config, plain):
     wrap_cols = ["Username"] if not plain else None
     persistent_table = CliTable(
         persistent_data,
-        title="Persistent Orgs",
+        title="Connected Orgs",
         wrap_cols=wrap_cols,
         bool_cols=["Default"],
     )
+    persistent_table.stringify_boolean_col(col_name="Default", true_str=CHECKMARK)
```

Comment:
What does this do?

Reply:
vestige of previous code design before rewrite